### PR TITLE
CRDCDH - 742

### PR DESCRIPTION
**Pull request by knockknockyoo**
throw an error for empty comment if admin role and meta/file error contained

### CODE REVIEW

File: verifier/submission-verifier.js [hunk 1]
```diff
@@ -47,13 +47,7 @@ class SubmissionActionVerifier {
...
     }
 
-    isValidRejectAction(comment) {
-        if(this.action === ACTIONS.REJECT && comment?.trim()?.length === 0) {
-            throw new Error(ERROR.VERIFY.INVALID_SUBMIT_ACTION);
-        }
-    }
-
-    isValidSubmitAction(role, aSubmission) {
+    isValidSubmitAction(role, aSubmission, comment) {
```
Comment (knockknockyoo) [inline verifier/submission-verifier.js:50]: Accidentally added from the previous PR. This is unused function.

File: verifier/submission-verifier.js [hunk 2]
```diff
@@ -66,6 +60,11 @@ class SubmissionActionVerifier {
                     throw new Error(ERROR.VERIFY.INVALID_SUBMIT_ACTION);
                 }
             }
+
+            const isError = [aSubmission?.metadataValidationStatus, aSubmission?.fileValidationStatus].includes(VALIDATION_STATUS.ERROR);
+            if ([ROLES.ADMIN].includes(role) && isError && (!comment || comment?.trim()?.length === 0)) {
```
Comment (vshand11) [inline verifier/submission-verifier.js:65]: change to ROLES.ADMIN === role

**Review by vshand11 — CHANGES_REQUESTED** [review]
a minor change.

File: verifier/submission-verifier.js (hunk 2 above)
Comment (knockknockyoo) [reply verifier/submission-verifier.js]: updated

**Review by vshand11 — APPROVED** [review]

**Pull request merged**